### PR TITLE
Blog-section

### PR DESCRIPTION
**Pull request by SingingApple**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -29,7 +29,7 @@ body .navbar-light .navbar-toggler-icon {
 }
 
 .avenir {
-  font-family: 'AvenirNext', serif;
+  font-family: "AvenirNext", serif;
```
Comment (rajat2502) [inline css/style.css]: Use singleQuote

File: css/style.css [hunk 2]
```diff
@@ -1279,6 +1279,9 @@ body .navbar-light .navbar-toggler-icon {
 .team-description h4 {
   padding: 0;
   font-size: 24px;
-  font-family: 'AvenirRegular';
+  font-family: "AvenirRegular";
 }
 /*# sourceMappingURL=style.css.map */
+#blog-container {
+  width: 90% !important;
```
Comment (rajat2502) [inline css/style.css:1265]: - First of all, don't use `important` if the same task can be done using other ways.
- The width of the cards looks odd
- Make the width & height of the blog cards the same as that of the club ones.
image

File: index.html [hunk 3]
```diff
@@ -326,7 +326,15 @@ <h4 class="mx-auto pt-4 avenirDemi" style="width: 90%">
...
-      <section id="about">
+      <!-- New blog section -->
+      <section id="club" style="background-color: #f9f9f9 !important">
+        <h1 class="text-center avenir">Blogs</h1>
+        <div class="container-fluid margin mx-auto pt-5" id="blog-container">
+          <div id="blog-text" class="row justify-content-between"></div>
+        </div>
+      </section>
+      <!--Blog section ends-->
+      <section id="about" style="background-color: #fff !important">
```
Comment (rajat2502) [inline index.html]: Don't use Inline CSS!

File: index.html [hunk 4]
```diff
@@ -326,7 +326,15 @@ <h4 class="mx-auto pt-4 avenirDemi" style="width: 90%">
           </div>
         </div>
       </section>
-      <section id="about">
+      <!-- New blog section -->
+      <section id="club" style="background-color: #f9f9f9 !important">
```
Comment (rajat2502) [inline index.html]: Inline!

File: js/blog.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  let postAuthor =
+    author.length > 22 ? `${author.substring(0, 19)}...` : author;
+  html = `
+  <a style="text-decoration: none; color: #000000" class="aveniDemi" href=${post.link} target="_blank">
+    <div class="col-md-3 pt-1 text-center" style="padding-left: 0; padding-right: 0;">
+      <div class="club_card justify-content-center mx-auto" style="margin-bottom: 15px; padding-bottom: 10px;">
+        <div class="card-img-top img-responsive mx-auto" style='width: 90%; height: 140px;  background: url("${post.image}"); background-size: cover; border-radius: 10px;' alt="Card image cap"/>
+      </div>
+      <div class="card-body" style="padding-bottom: 0px;" align="left">
+        <h4 class="avenirDemi" style="font-weight: 500; font-size: 24px; line-height: 33px;">${postTitle}</h4>
```
Comment (rajat2502) [inline js/blog.js]: Add a title attribute to this h4

File: js/blog.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
+//getting section for showing blog data
+const section = document.querySelector("#blog-text");
+let html = ``;
+//template
+const addTemplate = (post) => {
+  let { body, title, author } = post;
+  let postTitle = title.length > 33 ? `${title.substring(0, 30)}...` : title;
+  // let description = body.length > 30 ? `${body.substring(0, 50)}...` : body; //body truncate
```
Comment (rajat2502) [inline js/blog.js]: ?

File: js/blog.js [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+//     "link": "LINK",
+//     "image": "IMAGELINK",
+//      "authorImage":"author Image Link",
+//   }
+$.getJSON("../js/Blog.json", (json) => {
+  const arr = json.slice(-4);
+  arr.forEach((obj) => {
+    addTemplate(obj);
+  });
+});
```
Comment (rajat2502) [inline js/blog.js]: Convert into vanilla JS.

**Review by rajat2502 — COMMENTED** [review]
@SingingApple, make these changes and improve the design according to the that on Figma.

File: js/blog.js (hunk 6 above)
Comment (SingingApple) [reply js/blog.js]: For the **...** which appears if the title is too long.

**Pull request merged**